baybe/priors/basic.py

```diff
@@ -85,3 +90,25 @@ def _validate_order(self, _: Any, b: float) -> None:  # noqa: DOC101, DOC103
                 f"For {self.__class__.__name__}, the upper bound `b` (provided: {b}) "
                 f"must be larger than the lower bound `a` (provided: {self.a})."
             )
+
+
+@define(frozen=True)
+class BetaPrior(Prior):
+    """A beta prior parameterized by alpha and beta."""
+
+    alpha: float = field(converter=float, validator=gt(0.0))
```

Comment:
@AVHopp, @Scienfitz: Are we OK with the names here? I always find it a bit unfortunate that they are called `alpha` and `beta` when the distribution itself is called `beta`, but I guess that's the most common terminology?

Reply:
I'm fine with this
dont argue with wikipedia

Reply:
saw that, too. Not much better, I think :D

Reply:
well at least avoiding the name conflict (which was your issue with the names?) alpha and beta could still be mentioned in the docstrings

Reply:
Was about to change but then I saw: why the hell does torch give alpha the index 1 and beta the index 0!?! Isn't this super confusing, since alpha concentrates mass at 0 and beta at 1? Or am I missing something here?

Reply:
As discussed, `BetaPrior` will stay as class. And because non of the alternatives are any better, I'll leave with with the Wiki version and keep `alpha`/`beta`